Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth
# Closed #1425 winters

## Changelog:
- Implemented different climates.


# How to test:
Enter the game and check the province climates (it's visible when you hover the mouse over the province), tell your feedback.

Closes #1425.
